modules/oc-mirror-command-reference-v2.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+// * updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oc-mirror-command-reference-v2_{context}"]
+= Command reference for V2
+
+The following tables describe the `oc mirror` subcommands and flags for V2:
+
+.oc mirror subcommands
+[cols="1,2",options="header"]
+|===
+|Subcommand
+|Description
+
+|`completion`
+|Generate the autocompletion script for the specified shell.
+
+|`describe`
+|Output the contents of an image set.
+
+|`help`
+|Show help about any subcommand.
+
+|`version`
+|Output the oc-mirror version.
+
+|`delete`
+|Used to delete images in remote registry and local cache.
+
+|===
+
+.oc mirror flags
+[cols="1,2",options="header"]
+|===
+|Flag
+|Description
+
+|`-c`, `--config` `<string>`
+|Specify the path to an image set configuration file.
+
+|`--dest-skip-tls`
+|Disable TLS validation for the target registry.
+
+|`--dest-use-http`
+|Use plain HTTP for the target registry.
+
```

Comment:
To be tested if these work...
I know I have been using --dest-tls-verify=false with v2 instead

Reply:
Let me know which ones need to be added and which ones are not supported.